src/server/mailgun/mailGunHelpers.js

```diff
@@ -22,5 +22,26 @@ exports.sendMail = (req, res) => {
   });
 };
   
+exports.sendMailForArrival = (req, res) => {
```

Comment:
Need to send a success response back to the client on successful email sending, otherwise client might hang